man/git-ftp.1.md

```diff
@@ -565,6 +565,13 @@ to authenticate.
 	login john
 	password SECRET
 
+With git-ftp the credentials stored in this file are used if no username is set.
```

Comment:
I'm continuing Resmo's style of capitalising "Git-ftp" as a name. While I don't really mind which way we do it, I think we should be consistent and use Git-ftp here.